### DISCUSSION TITLE
Welcome to edgartools Discussions!

### DISCUSSION THREAD

**Discussion by dgunning** [dgunning/edgartools#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShipItAndPray** [comment]
I built **[Maintainer Shield](https://github.com/ShipItAndPray/maintainer-shield)** to help with this — a free, MIT-licensed GitHub Action that combines:

- **15-check AI slop PR detection** (AI language, commit timing, submission speed, author volume)
- **Issue auto-triage** (auto-labels bug/feature/question/docs, detects duplicates)
- **Contributor reputation scoring** (0-100 based on merge history, account age, activity)

All configurable. Collaborators auto-exempt. 5 lines to install.

https://github.com/ShipItAndPray/maintainer-shield
